Dockerfile

```diff
@@ -1,6 +1,9 @@
-#TODO
 FROM ubuntu:16.04
+LABEL description="Unity CSI Driver"
 
-COPY _output/csi-unity /csi-unity
+COPY ./_output/csi-unity /csi-unity
+
+RUN apt update
+RUN apt install -y nfs-common
```

Comment:
Put two `RUN` in one line, like `RUN apt update && apt install -y nft-common` to save one layer of Docker image.
And consider to run apt clean to make the image smaller.